### PR TITLE
fix: [CDS-53893]: fixed nexus3artifact trigger issues

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

#### [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

fixed the following prs
https://harness.atlassian.net/browse/CDS-53893
https://harness.atlassian.net/browse/CDS-53886
https://harness.atlassian.net/browse/CDS-53898

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger jest

**Comment by sahithikolichala** [comment]
retrigger coverage

File: src/modules/70-pipeline/utils/stageHelpers.ts [hunk 1]
```diff
@@ -94,10 +94,9 @@ const commonRepoFormatTypes = [
 
 export const nexus2RepositoryFormatTypes = [...commonRepoFormatTypes]
 
-export const nexus3RepositoryFormatTypes = [
-  ...commonRepoFormatTypes,
-  { label: 'Raw', value: RepositoryFormatTypes.Raw }
-]
+export const rawRepoFormat = [{ label: 'Raw', value: RepositoryFormatTypes.Raw }]
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/utils/stageHelpers.ts:97]: This label should comes from yaml

File: src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/NexusArtifact/NexusArtifact.tsx [hunk 2]
```diff
@@ -158,187 +160,224 @@ export function NexusArtifact({
...
+        {formik => {
+          return (
+            <Form>
+              <div className={css.artifactForm}>
+                <div className={css.imagePathContainer}>
+                  <FormInput.Select
+                    name="repositoryFormat"
+                    label={getString('common.repositoryFormat')}
+                    items={[...k8sRepositoryFormatTypes, ...nexus3RepositoryFormatTypes]}
+                    onChange={ev => {
```
Comment (pankajsingh-harness) [inline src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/NexusArtifact/NexusArtifact.tsx:172]: Re-confirm if onChange covers all the scenarios. As per [ui-core doc](https://uicore.harness.io/?path=/docs/form-select--basic)
```
onChange | ((item: SelectOption, event?: SyntheticEvent<HTMLElement, Event>) => void)

```

Comment (sahithikolichala) [reply src/modules/72-triggers/components/steps/ArtifactTriggerConfigPanel/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/NexusArtifact/NexusArtifact.tsx:172]: actually we dont need this change, onchange covers all the usecases!
In my local branch,i have removed this change

**Review by pankajsingh-harness — APPROVED** [review]

**Pull request merged**